website/src/components/IndexPage/getOrganismStatistics.ts

```diff
@@ -69,22 +70,22 @@ const getTotalAndLastUpdatedAt = async (
 const getRecent = async (organism: string, numberDaysAgo: number): Promise<number> => {
     const recentTimestamp = Math.floor(Date.now() / 1000 - numberDaysAgo * 24 * 60 * 60);
     const client = LapisClient.createForOrganism(organism);
-    const recentTotal = (
+    const recentTotalIncludingRevoked = (
         await client.call('aggregated', {
             [`${RELEASED_AT_FIELD}From`]: recentTimestamp,
             version: 1,
         })
     )
         .map((x) => x.data[0].count)
         .unwrapOr(0);
-    const recentRevoked = (
+    const recentTotalRevoked = (
         await client.call('aggregated', {
             [`${RELEASED_AT_FIELD}From`]: recentTimestamp,
             version: 1,
-            [VERSION_STATUS_FIELD]: siloVersionStatuses.revoked,
+            [VERSION_STATUS_FIELD]: versionStatuses.revoked,
```

Comment:
Ah - I see, technically things could be resubmitted and then we _should_ count them, but we aren't.

Because it would require a more complicated query that would require fetching accessions - correct?

We should clarify this in a comment.